qutebrowser/utils/standarddir.py

```diff
@@ -147,9 +147,12 @@ def _init_data(args: Optional[argparse.Namespace]) -> None:
     # system_data
     _locations.pop(_Location.system_data, None)  # Remove old state
     if utils.is_linux:
-        path = '/usr/share/' + APPNAME
-        if os.path.exists(path):
-            _locations[_Location.system_data] = path
+        if version.is_flatpak():
+            path = '/app/share/' + APPNAME
+        else:
+            path = '/usr/share/' + APPNAME
```

Comment:
```suggestion
        prefix = '/app' if version.is_flatpak() else '/usr'
        path = f'{prefix}/share/{APPNAME}'
```

might be a bit more readable?

Reply:
Thanks, it's now updated with the suggested changes.